common/src/main/java/org/wso2/carbon/testgrid/common/Script.java

```diff
@@ -46,20 +48,12 @@ public String toString() {
         }
     }
 
-    public ScriptType getScriptType() {
-        return scriptType;
-    }
-
-    public void setScriptType(ScriptType scriptType) {
-        this.scriptType = scriptType;
-    }
-
-    public String getFileName() {
-        return fileName;
+    public String getFilePath() {
```

Comment:
Better to include java doc comments since this would be referred to by the other developers.

Reply:
Will be added with #57.